js/core.js

```diff
@@ -681,6 +682,7 @@ SyncedStorage.defaults = {
     'profile_backpacktf': true,
     'profile_astatsnl': true,
     'profile_steamid': true,
+    'group_steamid': true,
```

Comment:
```suggestion
    'group_steamid': false,
```
Even more niche use case than the profile ID viewer imo